test/sql/copy/csv/test_csv_httpfs.test

```diff
@@ -353,3 +294,94 @@ select * from read_csv_auto('https://csvbase.com/meripaterson/stock-exchanges');
 249	North America	United States of America	Members' Exchange	NULL	2020-09-24
 250	Africa	Zimbabwe	Victoria Falls Stock Exchange	NULL	2020-11-01
 251	Asia	China	Beijing Stock Exchange	NULL	2021-12-27
+
+
+#FIXME this test fails: file is nonexistent
+mode skip
+
+query IIIIII rowsort
+SELECT * from read_csv_auto('https://github.com/duckdb/duckdb/raw/9cf66f950dde0173e1a863a7659b3ecf11bf3978/data/csv/customer.csv');
```

Comment:
Maybe its nice to use logging to ensure we get the expected http headers?

for example:
```
pragma enable_logging('HTTP');
-- do query
select response.headers['Connection'] from duckdb_logs_parsed('HTTP');
```

We could verify for example that the `__RESPONSE_STATUS__` header is properly parsed into the response headers: `select response.headers['__RESPONSE_STATUS__'] from duckdb_logs_parsed('HTTP');`